src/components/views/settings/tabs/PreferencesSettingsTab.js

```diff
@@ -0,0 +1,125 @@
+        'hideAvatarChanges', // TODO: Positive wording
+        'hideDisplaynameChanges', // TODO: Positive wording
+    ];
+
+    static ADVANCED_SETTINGS = [
+        'alwaysShowEncryptionIcons',
+        'Pill.shouldHidePillAvatar', // TODO: Positive wording
+        'TagPanel.disableTagPanel', // TODO: Positive wording
+        'promptBeforeInviteUnknownUsers',
+        // Start automatically after startup (electron-only)
+        // Autocomplete delay (niche text box)
+    ];
+
+    constructor() {
+        super();
+
+        this.state = {
+            autoLaunch: false,
+            autoLaunchSupported: false,
+        };
+    }
+
+    async componentWillMount(): void {
+        const autoLaunchSupported = await PlatformPeg.get().supportsAutoLaunch();
+        let autoLaunch = false;
+
+        if (autoLaunchSupported) {
+            autoLaunch = await PlatformPeg.get().getAutoLaunchEnabled();
+        }
+
+        this.setState({autoLaunch, autoLaunchSupported});
+    }
+
+    _onAutoLaunchChange = (checked) => {
+        PlatformPeg.get().setAutoLaunchEnabled(checked).then(() => this.setState({autoLaunch: checked}));
+    };
+
+    _onAutocompleteDelayChange = (e) => {
+        SettingsStore.setValue("autocompleteDelay", null, SettingLevel.DEVICE, e.target.value);
+    };
+
+    _renderGroup(settingIds) {
+        const SettingsFlag = sdk.getComponent("views.elements.SettingsFlag");
+        return settingIds.map(i => <SettingsFlag key={i} name={i} level={SettingLevel.ACCOUNT} />);
+    }
+
+    render() {
+        let autoLaunchOption = null;
+        if (this.state.autoLaunchSupported) {
+            autoLaunchOption = <LabelledToggleSwitch value={this.state.autoLaunch}
+                                                     onChange={this._onAutoLaunchChange}
+                                                     label={_t('Start automatically after system login')} />;
+        }
+
+        return (
+            <div className="mx_SettingsTab mx_PreferencesSettingsTab">
+                <div className="mx_SettingsTab_heading">{_t("Preferences")}</div>
+                <div className="mx_SettingsTab_section">
+                    <span className="mx_SettingsTab_subheading">{_t("Composer")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.COMPOSER_SETTINGS)}
+
+                    <span className="mx_SettingsTab_subheading">{_t("Room list")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.ROOM_LIST_SETTINGS)}
+
+                    <span className="mx_SettingsTab_subheading">{_t("Timeline")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.TIMELINE_SETTINGS)}
+
+                    <span className="mx_SettingsTab_subheading">{_t("Advanced")}</span>
+                    {this._renderGroup(PreferencesSettingsTab.ADVANCED_SETTINGS)}
+                    {autoLaunchOption}
+                    <Field id={"autocompleteDelay"} label={_t('Autocomplete delay (ms)')} type='number'
```

Comment:
Is that Nunito or some other font for the value inside the autocomplete input? (I believe the design wants Nunito.)

<img width="340" alt="2019-01-24 at 11 53" src="https://user-images.githubusercontent.com/279572/51697965-bf04d600-1fce-11e9-87b0-64af646f2dcd.png">

Reply:
Looks like it is indeed using Arial for some reason. 

/me shall fix

Reply:
Fixed 
![image](https://user-images.githubusercontent.com/1190097/51700782-38032e00-1fcd-11e9-9605-b1cfe5d0b841.png)